src/endpoint.ts

```diff
@@ -122,6 +124,7 @@ export class Endpoint<
   };
   protected readonly scopes: SCO[] = [];
   protected readonly tags: TAG[] = [];
+  protected readonly operationId?: string;
```

Comment:
there is a conflict due to the recent changes made in #1189 .
Please sync the `master` branch in your fork and update your PR.
The following property please rename the following way (entirely private):

```suggestion
  readonly #operationId?: string;
```